src/coreclr/jit/lsrabuild.cpp

```diff
@@ -911,15 +919,15 @@ regMaskTP LinearScan::getKillSetForBlockStore(GenTreeBlk* blkNode)
             if (isCopyBlk)
             {
                 // rep movs kills RCX, RDI and RSI
-                killMask = RBM_RCX | RBM_RDI | RBM_RSI;
+                killMask.AddGprRegs(RBM_RCX | RBM_RDI | RBM_RSI);
```

Comment:
Aren't `RBM_*` `regMaskTP` already? I don't really understand changes here (in multiple places it seems like we shouldn't need to switch to `SingleRegTypeMask` overloads)

Reply:
yes, all the `RBM_*` are still `uint64_t` except for `allAvailableRegs`. Here is what I am planning to do after this PR:
1. Send a PR that convert the `killMask`'s registerAssignment to `regMaskTP` (by unioning existing field or equivalent)
2. I am still deciding if we should convert all or just subset of `RBM_*`, but most likely I will convert just some `RBM_*` to `regMaskTP` that contains mask. All the remaining can continue to be `uint64_t`. In future, if we feel a need for TP improvement reason, we can convert them all to `regMaskTP` (once predicate register work is complete).